ogusa/scripts/execute.py

```diff
@@ -47,10 +49,15 @@ def runner(output_base, baseline_dir, test=False, time_path=True, baseline=False
         txfunc.get_tax_func_estimate(baseline=baseline, analytical_mtrs=analytical_mtrs, age_specific=age_specific,
                                      start_year=user_params['start_year'], reform=reform, guid=guid)
     print 'In runner, baseline is ', baseline
-    run_params = ogusa.parameters.get_parameters(test=test, baseline=baseline, guid=guid)
+    if small_open and (not isinstance(small_open, dict) or not all(key in small_open for key in SMALL_OPEN_KEYS)):
```

Comment:
@PeterDSteinberg You changes generally look good.  These lines of code have a lot going on and make things a bit opaque.  They look more complicated than what I thought would be the case for just adding one more user modifiable parameter, but I see that you are trying to have the boolean for "small open" and the world interest rate parameter together in just one argument and if that's the case I don't have an immediate suggestion of a cleaner way to do this.

Reply:
@jdebacker I simplified that part a bit, but it stands out mainly because it is the only parameter that is validated.